### DISCUSSION TITLE
Is there any way to expire perticular TestFlight Build using fastlane?

### DISCUSSION THREAD

**Discussion by nawaahme** [fastlane/fastlane#18205] [Q&A]
Is there any way to expire particular TestFlight Build Number using fastlane?

Currently, i'm able to expire previous upload TestFlight Build using Fastlane parameter "expire_previous_builds"

**Comment by dinsen** [comment]
Hi @nawaahme 
Yes, if Apple have not changed the endpoint for it, it should work with normal login and not the API Key.

If you have an object of `Spaceship::TestFlight::Build`you can use `expire!` on it.

```ruby
build = Spaceship::TestFlight::Build.find(...)
build.expire!
```
